src/inausoft.netCLI.Tests/Executor.Tests.cs

```diff
@@ -133,29 +133,33 @@ public void Executor_RunCLI_ShouldThrowInvalidCommmandException_ForEmptyArgument
             var config = new CLIConfiguration().Map<Command1, MockCommand1Handler>();
 
             //Act
-            Executor.RunCLI(config, args, mockCommand1Handler, mockCommand1Handler);
+            Assert.ThrowsException<InvalidCommandException>(() => Executor.RunCLI(config, args, mockCommand1Handler));
 
             //Assert with exception
```

Comment:
Act & assert is actually done above, this comment could be removed?

Reply:
I can adjust it to //Act & Assert.